From f18a70c85ebce8842da18ac0333b4d6db8f8479f Mon Sep 17 00:00:00 2001
From: Ansgar Burchardt <ansgar-guest@alioth.debian.org>
Date: Sat, 13 Dec 2008 15:00:01 +0100
Subject: [PATCH 8/8] handle SDL quit event

This patch makes Jump'n'bump quit when pressing the close button or
using other means to ask Jump'n'bump to quit (e.g. Alt+F4).

See http://bugs.debian.org/487333
---
 sdl/interrpt.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/sdl/interrpt.c b/sdl/interrpt.c
index 91a27ab..d6ab688 100644
--- a/sdl/interrpt.c
+++ b/sdl/interrpt.c
@@ -419,6 +419,9 @@ int intr_sysupdate()
 				break;
 			}
 			break;
+		case SDL_QUIT:
+			deinit_program();
+			break;
 		default:
 			break;
 		}
-- 
1.6.2.1

